### PR TITLE
Playlist Management

### PR DESCRIPTION
**Pull request by shrey27**
Added Features
- Create a playlist from videos listing page
- Delete any playlist from Playlists page in the sidebar
- Playlists page is protected

Not completed: CRUD operations for handling videos in a playlists

### CODE REVIEW

File: src/frontend/context/PlaylistContext.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { createContext, useContext, useReducer, useEffect } from 'react';
+import { useAuthCtx } from './index';
+import { getPlaylists, addPlaylist, deletePlaylist } from '../service';
+import { useLocalStorage } from '../helpers';
+// import { historyReducerFunction, historyDefaultState } from '../helpers';
```
Comment (romabulani) [inline src/frontend/context/PlaylistContext.js]: remove comments if not in use

File: src/frontend/context/PlaylistContext.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      dispatch({ type: 'PLAYLIST_API_RESPONSE', payload: [...playlistsArray] });
+    }
+  };
+
+  useEffect(() => {
+    const getPlaylistsFunction = async () => {
+      dispatch({ type: 'PLAYLIST_API_REQUEST' });
+
+      const playlistsArray = await getPlaylists(token);
+      console.log('playlistsArray', playlistsArray);
```
Comment (romabulani) [inline src/frontend/context/PlaylistContext.js]: Remove console logs

File: src/frontend/context/PlaylistContext.js (hunk 1 above)
Comment (Shah-Arsalan) [inline src/frontend/context/PlaylistContext.js]: remove the comment

**Review by lomesshh — APPROVED** [review]
Clean and modular code 
Looks good to merge

**Review by romabulani — APPROVED** [review]
- Code and folder structure looks good

Suggestions : 
- Have the check for creating playlist with same name again
- Also, Playlist with empty name or only spaces should not be created

**Review by Shah-Arsalan — APPROVED** [review]
Code looks clean . Nothing really that bothered me . You can merge

File: src/frontend/context/PlaylistContext.js (hunk 1 above)
Comment (shrey27) [reply src/frontend/context/PlaylistContext.js]: that's a good suggestion. I will put empty check also

**Pull request merged**